README.md

```diff
@@ -88,6 +88,26 @@ export default Component.extend({
 });
 ```
 
+### Segment ready
+
+You can wait for segment readyness:
+
+```js
+// app/components/some-awesome-component.js
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
```

Comment:
Could you replace this example with a glimmer component using native classes?

Here is the sniped you could use:

```js
import Component from '@glimmer/component';
import { inject as service } from '@ember/service';

export default class MyComponent extends Component {
  @service segment;

  constructor() {
    super(...arguments);

    this.segment.ready(() => {
      console.log('segment is ready');
    });
  }
}
```

Reply:
Fix in d50efa51325fda533c15efd37b5e4c7682060bc0